src/zentables/zentables.py

```diff
@@ -819,14 +827,101 @@ def _swap_column_levels(df: pd.DataFrame) -> pd.DataFrame:
 
 
 def _combine_n_pct(
-    df_n: pd.DataFrame, df_pct: pd.DataFrame, digits: int = 1
+    df_n: pd.DataFrame,
+    df_pct: pd.DataFrame,
+    digits: int = 1,
+    suppress_symbol: str = "*",
 ) -> pd.DataFrame:
     """
     Helper function that formats and combines n and pct values
     """
     df_n_str = df_n.astype(str)
     df_pct_str = df_pct.applymap(lambda x: f" ({x:.{digits}%})", na_action="ignore")
-    return df_n_str.add(df_pct_str)
+    return df_n_str.add(df_pct_str).fillna(f"{suppress_symbol}")
+
+
+def _is_between(x, ceiling: int = 10):
+    """
+    Helper function that verifies if the input is below ceiling value
+    """
+    return x in range(1, ceiling)
+
+
+def _add_random_noise(x, rng):
+    """
+    Helper function that adds random noise to entry according a Uniform distribution
+    """
+    return x + rng.uniform(0, 1)
+
+
+def _local_suppression(mini_df_n: pd.DataFrame, ceiling: int = 10, seed: int = 2021):
+    """
+    Helper function that applies cell suppression to
+    """
+    rng = np.random.default_rng(seed)
```

Comment:
To say, it's common that you want to _share_ an rng between functions. In that case, you may wish to add a utility function that looks something like:

```python
def _seed_to_rng(seed: Optional[Union[int, Generator]] = None) -> Generator:
    if seed is None:
        return np.random
    if isinstance(seed, int):
        return np.random.default_rng(seed)
    return seed
```